runtime/service_integ_test.go

```diff
@@ -247,6 +250,86 @@ func createTapDevice(ctx context.Context, tapName string) error {
 	return nil
 }
 
+func TestMemoryBalloon_Isolated(t *testing.T) {
+	prepareIntegTest(t)
+	ctx := namespaces.WithNamespace(context.Background(), defaultNamespace)
+
+	pluginClient, err := ttrpcutil.NewClient(containerdSockPath + ".ttrpc")
+	require.NoError(t, err, "failed to create ttrpc client")
+
+	numVMs := 5
+	var vmGroup sync.WaitGroup
+	for i := 0; i < numVMs; i++ {
```

Comment:
Why do you want to have 5 VMs?

Reply:
> Are they nil-able on Firecracker's schema? I agree it is better to check that before de-referencing, but this code only logs error after that. You must not de-reference them if they are nil."

Nope, they aren't nil-able. But I still think it is a good practice to leave this safe deference check, I just changed the log to return an error. 

> Why do you want to have 5 VMs?
It is an arbitrary number. I just wanted to test if it works with more than 1 VMs.